website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -1848,6 +1848,169 @@ $ curl \
 }
 ```
 
+## Assess a workspace
+
+This endpoint assesses a workspace. Only users with admin access are authorized to assess a workspace through a browser session.
```

Comment:
could you clarify why we added that `admin access` sentence? I'm not too sure what we mean by this

Reply:
So at the pundit policy level, we gate this endpoint by [mutate?](https://github.com/hashicorp/atlas/blob/main/app/policies/workspace_pundit_policy.rb#L171) permissions which are admin level.

Reply:
That makes sense. I think the admin wording confused me since it is used so ubiquitously, but if there is context then we should be good